server.js

```diff
@@ -1,35 +1,105 @@
-import express from "express";
-import cors from "cors";
-import mongoose from "mongoose";
+import express from "express"
+import cors from "cors"
+import mongoose from "mongoose"
 
 // If you're using one of our datasets, uncomment the appropriate import below
 // to get started!
 // import avocadoSalesData from "./data/avocado-sales.json";
 // import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
+// import goldenGlobesData from "./data/golden-globes.json"
+import netflixShows from "./data/netflix-titles.json"
+//import { all } from "express/lib/application"
 // import topMusicData from "./data/top-music.json";
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
-mongoose.Promise = Promise;
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
+mongoose.Promise = Promise
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
+const port = process.env.PORT || 8080
+const app = express()
+
+const Netflixshow = mongoose.model("Netflixshow", {
+  show_id: Number,
+  title: String,
+  director: String,
+  cast: String,
+  country: String,
+  date_added: String,
+  release_year: Number,
+  rating: String,
+  duration: String,
+  listed_in: String,
+  description: String,
+  type: String,
+})
+
+if (process.env.RESET_DB) {
+  const seedDatabase = async () => {
+    await Netflixshow.deleteMany()
+    netflixShows.forEach((singleshow) => {
+      const newShow = new Netflixshow(singleshow)
+      newShow.save()
+    })
+  }
+  seedDatabase()
+}
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
+app.use(cors())
+app.use(express.json())
 
-// Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
+  res.send("Hello! This is the API for week 18!!")
+})
+
+app.get("/netflixshows/director/:director", async (req, res) => {
+  const singleDirector = await Netflixshow.findOne({
+    director: req.params.director,
+  })
+  res.status(200).send(singleDirector)
+})
+
```

Comment:
You could consider adding try and catching error message / so you will have better backend because you get error messages and know what is wrong if is is not finding a director with given value

Reply:
You can added this to every endpoints